cmd/traffic/cmd/manager/manager.go

```diff
@@ -152,6 +195,21 @@ func (s *service) servePrometheus(ctx context.Context) error {
 		return int(atomic.LoadInt32(&s.activeGrpcRequests))
 	})
 
+	labels := []string{"client", "install_id"}
+	InterceptCounter = newCounterVecFunc("intercept_count", "The total number of global intercepts by user", labels)
+	InterceptActiveStatusGauge = newGaugeVecFunc("intercept_active_status",
+		"Flag to indicate when an intercept is active. 1 for active, 0 for not active.", append(labels, "workload"))
+	ConnectCounter = newCounterVecFunc("connect_count", "The total number of connects by user", labels)
+	ConnectActiveStatusGauge = newGaugeVecFunc("connect_active_status", "Flag to indicate when a connect is active. 1 for active, 0 for not active.", labels)
+
+	s.state.SetConnectionEstablishedFinalizer(func(client, installId string) {
```

Comment:
The use of the term "connection" is somewhat ambiguous and the term "established" here seems a bit redundant. Could this be named `SetAllClientSessionsFinalizer`? Perhaps pass the `*rpc.ClientInfo` instead of passing its name and install-id explicitly. Less to change, should we want to use some other info.